iiblib/iib_build_details_model.py

```diff
@@ -61,11 +61,37 @@ class IIBBuildDetailsModel(object):
 
     _operation_attrs = []
 
+    _accepts = ""
```

Comment:
What about longer name `_accepts_request_type = ""` or at least putting comment above? (personally I vote for longer name so those who inherits it knows what the variable is)

Reply:
Chnaged